.github/deploy.sh

```diff
@@ -64,19 +69,9 @@ ENCRYPTED_KEY=${!ENCRYPTED_KEY_VAR}
 ENCRYPTED_IV=${!ENCRYPTED_IV_VAR}
 openssl aes-256-cbc -K "$ENCRYPTED_KEY" -iv "$ENCRYPTED_IV" -in .github/deploy_key.enc -out .github/deploy_key -d
 chmod 600 .github/deploy_key
-eval $(ssh-agent -s)
+eval "$(ssh-agent -s)"
```

Comment:
(Why) Is this necessary?

Reply:
Not a functional change, but let's compare the result:
```console
% echo $(ssh-agent -s )
SSH_AUTH_SOCK=/tmp/ssh-uekMnQ1X5cAJ/agent.30019; export SSH_AUTH_SOCK; SSH_AGENT_PID=30020; export SSH_AGENT_PID; echo Agent pid 30020;
% echo "$(ssh-agent -s )"
SSH_AUTH_SOCK=/tmp/ssh-2LsmK7aT6yYE/agent.30210; export SSH_AUTH_SOCK;
SSH_AGENT_PID=30211; export SSH_AGENT_PID;
echo Agent pid 30211;
```